MasterSetup.sql

```diff
@@ -42,7 +42,8 @@ CREATE INDEX column_info_idx
 CREATE FUNCTION dbmirror2.recordchange()
 RETURNS trigger AS $$
 DECLARE
-    tablename   TEXT;
+    _tablename  TEXT;
+    -- prefixed with an underscore to disambiguate it from the column name
```

Comment:
Separate comment line usually comes first:

```suggestion
    -- prefixed with an underscore to disambiguate it from the column name
    _tablename  TEXT;
```

Reply:
I don’t really get what it is distinguishing to. A reference might be useful.

Reply:
A query such as `select * from artist where id = id` is valid sql in that it would select all rows.  Now suppose you're writing this in pl/pgsql and also have a variable named `id`.  It's ambiguous whether you're referencing the variable `id` or the column `id`.

Reply:
Here's an example:

```
musicbrainz_test=> DO $$
DECLARE
    id integer;
BEGIN
    id := 1;
    select * from artist where id = id;
END $$;
ERROR:  column reference "id" is ambiguous
LINE 1: select * from artist where id = id
                                   ^
DETAIL:  It could refer to either a PL/pgSQL variable or a table column.
QUERY:  select * from artist where id = id
CONTEXT:  PL/pgSQL function inline_code_block line 6 at SQL statement
```